frontend/reactapp/src/pages/ProductPage/productPage.css

```diff
@@ -0,0 +1,159 @@
+.bg-container {
```

Comment:
Can we use just Product instead of ProductPage ? 
Its clear name for page with "Product" only.
This convention will be good when you have a long name for pages or components.